docs/BreakingChanges/141 WPF TreeViewItem must be used within a TreeView.md

```diff
@@ -0,0 +1,27 @@
+## ID: WPF TreeViewItem must be used within a TreeView
+
+### Scope
+Major
+
+### Version Introduced
+4.5
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+Before 4.5, a TreeViewItem could be used outside of a TreeView. A change in 4.5 added a check during scrolling that assumes TreeViewItems is only used in the context of 
+TreeView. An example of when this is hit is if a TreeViewItem is the root of a DataTemplate or is used without being wihtin a TreeView.
```

Comment:
line 14 'wihtin' -> 'within'.

The bug only shows up when
a. TreeViewItem's visual parent is not a panel.   (A TVI generated for a TreeView will have a panel as its parent)
b. The TVI is a descendant of a VirtualizingStackPanel acting as the "items host" for a list control (ListBox, DataGrid, ListView, etc.).   Virtualization doesn't need to be enabled.
c. The VSP is item-scrolling (ScrollUnit="Item").
d. Someone calls VSP.MakeVisible(v) to scroll an element v into view.   This can be done explicitly, or implicitly in a number of ways;  perhaps the most common way is simply clicking on v to give it the keyboard focus.
e. The visual-parent chain from v to the VSP passes through the TVI.

In other words, not only do you need to use a TreeViewItem outside of a TreeView, you also need the user to click on a descendant of the TVI to bring it into view.   If the TVI has no focusable descendants, you'll never see the bug.